src/overlays/actors/ovl_Eff_Lastday/z_eff_lastday.h

```diff
@@ -3,16 +3,36 @@
 
 #include "global.h"
 
+#define EFFLASTDAY_GET_F(thisx) ((thisx)->params & 0xF)
+
 struct EffLastday;
 
 typedef void (*EffLastdayActionFunc)(struct EffLastday*, PlayState*);
 
 typedef struct EffLastday {
-    /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x10];
-    /* 0x0154 */ EffLastdayActionFunc actionFunc;
+    /* 0x000 */ Actor actor;
+    /* 0x144 */ Gfx* dList;
+    /* 0x148 */ AnimatedMaterial* matAnim;
+    /* 0x14C */ s16 step;
+    /* 0x14E */ s16 alpha;
+    /* 0x150 */ s16 actionCmd;
+    /* 0x154 */ EffLastdayActionFunc actionFunc;
 } EffLastday; // size = 0x158
 
+typedef enum EffLastDayAction {
+    /* 0x0 */ ACTION_NONE,
```

Comment:
```suggestion
    /* 0x0 */ ACTION_0,
```
Personally I don't think you should use `NONE` here (same for Params below) as `NONE` implies that no action is going on (which might be happening here, but it is hard to tell without more documentation. I think just using 0 is fine for now.